README.md

```diff
@@ -1,4 +1,4 @@
-## File Indexer [![Build Status](https://travis-ci.org/farhaanbukhsh/file-indexer.svg?branch=master)](https://travis-ci.org/farhaanbukhsh/file-indexer)
+## File Indexer [![Build Status](https://travis-ci.org/farhaanbukhsh/dexer.svg?branch=master)](https://travis-ci.org/farhaanbukhsh/dexer)
```

Comment:
Missed to change `farhaanbukhsh` to `dgplug`

Reply:
I left that knowingly. Seems like that also got upgraded.